precompiles/avs/abi.json

```diff
@@ -23,9 +81,73 @@
 			},
 			{
 				"internalType": "uint64",
-				"name": "action",
+				"name": "minSelfDelegation",
 				"type": "uint64"
 			},
+			{
+				"internalType": "uint64",
+				"name": "unbondingPeriod",
+				"type": "uint64"
+			},
+			{
+				"internalType": "string",
+				"name": "epochIdentifier",
+				"type": "string"
+			}
+		],
+		"name": "RegisterAVS",
+		"outputs": [
+			{
+				"internalType": "bool",
+				"name": "success",
+				"type": "bool"
+			}
+		],
+		"stateMutability": "nonpayable",
+		"type": "function"
+	},
+	{
+		"inputs": [
+
+		],
+		"name": "RegisterOperatorToAVS",
```

Comment:
Same here, can be into unformed into RegisterOrDeregisterOperator? And then check afterwards in the backend?